### PR TITLE
docs: add new delegates search params

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

Adds the new delegate search body params.

## Types of changes

- [x] Other... Please describe: docs

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

File: docs/api/public/v2/delegates.md [hunk 1]
```diff
@@ -312,17 +312,45 @@ POST /api/delegates/search
...
 | Name  | Type | Description                                   | Required |
 |-------|:----:|-----------------------------------------------|:--------:|
 | page  | int  | The number of the page that will be returned. | :x:      |
 | limit | int  | The number of resources per page.             | :x:      |
 
+### Body Parameters
+
+| Name                      | Type   | Description  | Required |
+|---------------------------|:------:|--------------|:--------:|
+| orderBy                   | string | ...          | :x:      |
```
Comment (faustbrian) [inline docs/api/public/v2/delegates.md]: Could you replace `...` with a few words of what each parameter is.

**Review by faustbrian — CHANGES_REQUESTED** [review]
Minor changes

**Review by faustbrian — APPROVED** [review]

**Comment by ghost** [comment]
A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $10 USD._

**Pull request merged**